test/e2e/snaps/ppom-toggle-settings.spec.js

```diff
@@ -0,0 +1,75 @@
+const { strict: assert } = require('assert');
+const {
+  withFixtures,
+  unlockWallet,
+  openDapp,
+  defaultGanacheOptions,
+  getWindowHandles,
+} = require('../helpers');
+const FixtureBuilder = require('../fixture-builder');
+
+describe('PPOM Settings', function () {
+  it('should not show the PPOM warning when toggle is off', async function () {
+    await withFixtures(
+      {
+        dapp: true,
+        fixtures: new FixtureBuilder()
+          .withPermissionControllerConnectedToTestDapp()
+          .build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test.title,
+      },
+      async ({ driver }) => {
+        await driver.navigate();
+        await unlockWallet(driver);
+
+        await driver.clickElement(
+          '[data-testid="account-options-menu-button"]',
+        );
+
+        await driver.clickElement({ text: 'Settings', tag: 'div' });
+        await driver.clickElement({ text: 'Experimental', tag: 'div' });
+
+        await openDapp(driver);
+        await driver.clickElement('#maliciousPermit');
```

Comment:
after triggering the transaction, we need to check the MetaMask popup and add an assertion to check that the PPOM Warning is not there